### ISSUE TITLE
Poster setup

### ISSUE THREAD

**Issue by JMitnik** [JMitnik/NLP-Posterior-Collapse-and-Probing#5]
## Introduction
Language models and text generation, RNNS are good and Sentence-VAEs are the extension of this. 
Sentence-VAE short explanation: how it works (reliance on both the encode and previous context)

VAEs are better, but have a problem -> Point to loss function, mark the reconstruction loss and KL loss in this formula
* Note that we don't want the KL loss to go to zero, as this effectively means that the decoder receives information that is roughly equal to nothing.

So our question and mission:
- Can we introduce means to increase this KL?

## Methods
We use a Recurrent Neural Network as our baseline to ensure that anything, we ensure that our VAE can't regress in terms of performance beyond the RNN.

We use multiple methods to try to counter this [Make one figure to show all three methods]:
1. Word-dropout: decoder is forced to rely on the latent because some of the incoming words are masked.
2. Free-bits: a hinge loss which essentially tries to push the KL to be above a certain number
3. Mu-forcing: a loss which ensures that the multivariate means of the posterior becomes more diverse, acting like an opposing magnet trying to push the posterior's means away from one another (and thus also away from the prior means of 1).

## Experimental setup
Network-specific hyperparameters are used from Bowman's paper, including embedding size, LSTM sizes, etc.

We use the PennTree Dataset, where we split our training, validation and testing in [x/x/x] percentages. Our task is to predict the next word, for each word in a target sentence. We do this for a batch size of 32 due to computing constraints.

For metrics, we capture the (negative) log likelihood (for VAE, we base this likelihood based on an MC-sample of 10), perplexity, and KL loss.

## Results

[Show table of hyperparameters?]
[Show three of the training plots from cross-validation: the NLLLoss, KLLoss and the Mu-Forcing (?)]
 
What we can say about these:
* [Model with the lowest NLL-Loss in total (which parameters)?]
* [Model with the highest KL-loss (which parameters)] -> Implication?
* [Model with the highest 

We compare in total 6 models on the test-set: the RNNLM, the VAE with best parameters fixed and variant parameters from hyperparameter selection, and the VAEs with one of these methods each applied, each with the best hyperparameter setup as indicated during hyperparameter tuning

[Show results table]

What we notice is clear-cut:
* [What can we say about the lowest NLL-Loss in general]
* [Which method presents the highest KL-loss]
* [If we choose to include MU figure]

## Discussion
?
